electron/js/menu/system.js

```diff
@@ -28,6 +30,11 @@ const locale = require('./../../locale/locale');
 let menu;
 var menuTemplate;
 
+const launcher = new autoLaunch({
+  name: 'Wire',
```

Comment:
just import `const pkg = require('./../../package.json');` and then use the `pkg.productName`